patches/picolibc.patch

```diff
@@ -32,3 +32,56 @@ index c757f7665..e262e1c21 100644
  
  	.bss (NOLOAD) : {
  		*(.sbss*)
+diff --git a/picolibc.ld.in b/picolibc.ld.in
```

Comment:
This is a duplicate of line 20 of the patch file and it prevents the patch from applying cleanly. While running `git apply` I get `error: patch failed: picolibc.ld.in:207`

Reply:
fixed.

Reply:
Please ensure that the code has actually been changed in the pull request before commenting that something has been fixed. Maybe in your local copy you've fixed it, but as far as everyone else is concerned the error remains and your comment is incorrect.